### ISSUE TITLE
 Project.getConvention() method has been deprecated

### ISSUE THREAD

**Issue by markwoon** [gradle-nexus/publish-plugin#281]
After updating to gradle > 8.2 I'm getting:
```
The Project.getConvention() method has been deprecated. This is scheduled to be removed in Gradle 9.0.
Consult the upgrading guide for further information:
https://docs.gradle.org/8.4/userguide/upgrading_version_8.html#deprecated_access_to_conventions
```

Any chance of an update for this?

**Comment by TWiStErRob** [comment]
Duplicate of #226, try the RC.

**Comment by remal** [comment]
@TWiStErRob are there any plans regarding stable v2 releave?

**Comment by szpak** [comment]
I believe that having ~3 things to sort out (#254/#171 - it's blocked on me :-/ and #274/#258) we could do it. @TWiStErRob Do you see anything else important for 2.0.0-final?

**Comment by szpak** [comment]
I've just released [2.0.0-rc-2](https://github.com/gradle-nexus/publish-plugin/releases/tag/v2.0.0-rc-2). Do you see anything else to be included in 2.0.0-final?

**Comment by TWiStErRob** [comment]
Closing as duplicate of https://github.com/gradle-nexus/publish-plugin/issues/226
